### PR TITLE
[CI:DOCS] [Release Notes]: add static routes

### PR DESCRIPTION
**Pull request by Cydox**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by lsm5** [comment]
@Luap99 @ashley-cui @mheon PTAL at this one. I closed mine

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 - The `--ip-range` option to the `podman network create` command now accepts a new syntax, `<startIP>-<endIP>`, which allows more flexibility when limiting the ip range that Podman assigns.
 - [Tech Preview] A new command, `podmansh`, has been added, which executes a user shell within a container when the user logs into the system. The container that the users get added to can be defined via a Podman Quadlet file. This feature is currently a `Tech Preview` which means it's ready for users to try out but changes can be expected in upcoming versions.
 - The `podman network create` command supports a new `--option`, `bclim`, for the `macvlan` driver.
+- The `podman network create` command now supports adding static routes using the `--route` flag.
```
Comment (Luap99) [inline RELEASE_NOTES.md:12]:
```suggestion
- The `podman network create` command now supports adding static routes using the `--route` option.
```
We try to only use the term option in docs.

**Review by ashley-cui — COMMENTED** [review]
/lgtm
/hold

**Comment by mheon** [comment]
Sure, LGTM

**Pull request merged**